Scoop/Controller/Onboarding/PhoneNumberViewController.swift

```diff
@@ -29,40 +29,54 @@ class PhoneNumberViewController: OnboardingViewController {
             }
             
             Networking.shared.currentUser.phoneNumber =  phoneNumber
-            
             self.delegate?.didTapNext(navCtrl, nextViewController: nil)
         }
-       
+        
+        setupTitleLines()
+        backButton.isHidden = true
         setupStackView()
         setupNextButton(action: nextAction ?? UIAction(handler: { _ in
             return
         }))
     }
     
     private func setupStackView() {
+        let textFieldBorderWidth = 1.0
+        let textFieldCornerRadius = 8.0
+        let textFieldBorderColor = UIColor(red: 0.584, green: 0.616, blue: 0.647, alpha: 1).cgColor
+        let leadingTrailingInset = 20
         stackView.axis = .vertical
         stackView.distribution = .fill
-        stackView.spacing = 20
+        stackView.spacing = 12
         stackView.alignment = .leading
         view.addSubview(stackView)
 
         stackView.snp.makeConstraints { make in
-            make.leading.trailing.equalToSuperview().inset(60)
+            make.leading.trailing.equalToSuperview().inset(leadingTrailingInset)
             make.center.equalToSuperview()
         }
         
         let titleLabel = UILabel()
         titleLabel.font = .systemFont(ofSize: 22)
-        titleLabel.text = "Phone Number"
+        titleLabel.text = "PHONE NUMBER"
```

Comment:
You might need to add an `accessibilityLabel` here. Since if someone is using the app with a voice assistant, text in all caps will be read out character by character, as opposed to just the text.